front/src/app/components/volunteer-card/volunteer-card.component.html

```diff
@@ -1,49 +1,32 @@
-<app-modal-aplication
-  *ngIf="openModalQuestion"
-  [idOrg]="idOrganization"
-  [idVol]="idVolunteering"
-  [nameVol]="nameVol"
-  [nameOrg]="nameOrg"
-  (onModalAfterApplication)="onModalQuestionAftertApplication($event)"
-  (closeModalQuestion)="closeModalApplication()"
->
+<app-modal-aplication *ngIf="openModalQuestion" [idOrg]="idOrganization" [idVol]="idVolunteering" [nameVol]="nameVol"
+  [nameOrg]="nameOrg" (onModalAfterApplication)="onModalQuestionAftertApplication($event)"
+  (closeModalQuestion)="closeModalApplication()">
 </app-modal-aplication>
 
-<app-modal-after-application
-  *ngIf="afterApplication"
-  [status]="statusModalApplication"
-  [message]="messageModalApplication"
-  (continueViewing)="continueViewing()"
->
+<app-modal-after-application *ngIf="afterApplication" [status]="statusModalApplication"
+  [message]="messageModalApplication" (continueViewing)="continueViewing()">
 </app-modal-after-application>
 <article
-  class="w-full flex flex-col items-center justify-center shadow shadow-lightGray p-4 gap-4 rounded-lg sm:flex-row lg:p-6"
->
+  class="w-full flex flex-col items-center justify-center shadow shadow-lightGray p-4 gap-4 rounded-lg sm:flex-row lg:p-6">
   <figure class="w-[100px] sm:w-[200px]">
-    <img
-      [src]="
+    <img [src]="
         data.organization?.image
           ? data.organization?.image.imageUrl
           : '/assets/imgs/not-found.webp'
-      "
-      alt=""
-      class="w-full h-full object-contain"
-    />
+      " alt="" class="w-full h-full object-contain" />
   </figure>
```

Comment:
Alternativa

<img [src]="data.organization?.image?.imageUrl || '/assets/imgs/not-found.webp'"
      class="w-full h-full object-contain" 
      alt="" class="w-full h-full object-contain" />